### DISCUSSION TITLE
How to use process groups?

### DISCUSSION THREAD

**Discussion by RudolfVonKrugstein** [gleam-lang/gleam#3118] [Questions & support] [answered]
I want to have some functionality, like phoenix pubsub for broadcasting.

It seems to me, that [process groups](https://www.erlang.org/doc/man/pg.html) are the functionality from erlang one needs. I did not find any reference for gleam and process groups on the internet. Am I missing something? should I just write ffis and all is good?

**Comment by lpil** [comment, marked as answer]
Yup! Everything works the same way in Gleam as in Erlang.
